.env

```diff
@@ -1,2 +1,9 @@
 API_BASE_URL=http://localhost:8000/api/v1
-NEXT_PUBLIC_BASE_URL=http://localhost:8000/api/v1
\ No newline at end of file
+NEXT_PUBLIC_BASE_URL=http://localhost:8000/api/v1
+NEXT_PUBLIC_FIREBASE_API_KEY=AIzaSyD4JsLvRzPjdBxwL3ambFze5jIWxsrIOWA
+NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=xplorers-website.firebaseapp.com
+NEXT_PUBLIC_FIREBASE_PROJECT_ID=xplorers-website
+NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=xplorers-website.appspot.com
+NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=168253186789
+NEXT_PUBLIC_FIREBSE_APP_ID=1:168253186789:web:330d78043386730a93aa0c
+NEXT_PUBLIC_FIREBSE_MEASUREMENT_ID=G-PZ530J9JZ5
```

Comment:
@Aaasisss Please create another project in Firebase and delete this project asap.